README.md

```diff
@@ -1,12 +1,5 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+This repo contains Fareeda's portfolio and SPS projects.
 
-To get started:
-
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/google/software-product-sprint.git`
-- Open the GitHub setup walkthrough: `teachme ~/software-product-sprint/walkthroughs/week-0-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
-
-Then follow the on-screen instructions to set up your repo.
+Enjoy looking through all the amazing content!
```

Comment:
Nit: Since it is a repo readme I will be more specific.
For example:
```suggestion
- Personal portfolio made with HTML, JS, CSS hosted on gcloud.
```

You can change the `readme` as we add more and more to your repo during the project